iogt_users/factories.py

```diff
@@ -20,7 +20,7 @@ class Meta:
 class AdminUserFactory(DjangoModelFactory):
```

Comment:
Seems like an admin user is just a 'normal' user with `is_superuser = True`.

To avoid duplication, we could either make `AdminUserFactory` extend `UserFactory`:
```
class AdminUserFactory(UserFactory):
    is_superuser = True
```

Or, just set the superuser flag when creating a normal user:
```
admin_user = UserFactory(is_superuser = True)
```